src/main/java/org/biouno/unochoice/util/SafeHtmlExtendedMarkupFormatter.java

```diff
@@ -0,0 +1,47 @@
+package org.biouno.unochoice.util;
+
+import com.google.common.base.Throwables;
+import hudson.markup.BasicPolicy;
+import hudson.markup.MarkupFormatter;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.HtmlSanitizer;
+import org.owasp.html.HtmlStreamRenderer;
+import org.owasp.html.PolicyFactory;
+
+import java.io.IOException;
+import java.io.Writer;
+
+public class SafeHtmlExtendedMarkupFormatter extends MarkupFormatter {
```

Comment:
There's really no reason to make this a markup formatter, as it's only used statically in this plugin.

Reply:
That's good news @daniel-beck ! I thought we would have to make a global markup formatter, available to other plugins, which would let users use it in other jobs or with other plugins. Simpler the way it is I guess :+1: 

I think that means the PR as-is should be good to go @apottere. Will prepare the changelog/docs/update issue later and merge it :-) thanks!

Reply:
Yes, without `@Extension` it doesn't matter much, except potential confusion years in the future.

Should also be part of plugin docs, I would consider it rather likely that this will drift further from other markup formatters' behavior due to changes in this plugin or in antisamy-markup-formatter.

Reply:
@kinow do you want to change this or should I?

Reply:
I believe we don't have to change anything @apottere . We will go with your original code that adds a formatter used only within the plug-in. I will update the documentation, and will include a note about it when tweeting the new release :) :+1: